src/net.cpp

```diff
@@ -2813,6 +2813,8 @@ void CConnman::PushMessage(CNode* pnode, CSerializedNetMsg&& msg)
 
         if (pnode->nSendSize > nSendBufferMaxSize)
             pnode->fPauseSend = true;
+        if (gArgs.IsArgSet("-logmessages"))
```

Comment:
nit:

```suggestion
        if (gArgs.IsArgSet("-logmessages")) {
```

Reply:
Ah I misread the style guide!

Now that I'm adding braces to this, I'll add them to the rest of PushMessage.